README.md

```diff
@@ -49,14 +49,14 @@ Here is a short video from @cpw showing how to install and setup Forge
 ## Pull requests
 
 Pull requests should target the current default branch. Currently, that is 
- the 1.10.x branch for Minecraft 1.10.2.
+ the 1.12.x branch for Minecraft 1.12.2.
 
 If you intend to modify a minecraft patch, or add a new one, always run
 ```gradlew genPatches``` to generate your patch and validate that you
 are not doing anything unexpected in your patch.
 
 ### Contributor License Agreement
-As stated in the (https://github.com/MinecraftForge/MinecraftForge/blob/1.10.x/LICENSE-new.txt)
+As stated in the (https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/LICENSE-new.txt)
```

Comment:
You should only resolve one issue per PR.

Reply:
I thought that doing one PR might be overkill for just changing to numbers. I'll split them up.

Reply:
See https://github.com/MinecraftForge/MinecraftForge/issues/4618.